addons/mail/static/src/views/web/fields/html_mail_field/html_mail_field.js

```diff
@@ -6,14 +6,23 @@ import { ColumnPlugin } from "@html_editor/main/column_plugin";
 const cssRulesByElement = new WeakMap();
 
 export class HtmlMailField extends HtmlField {
-    async getEditorContent() {
-        if (!cssRulesByElement.has(this.editor.editable)) {
-            cssRulesByElement.set(this.editor.editable, getCSSRules(this.editor.document));
+    /**
+     * @param {WeakMap} cssRulesByElement
+     * @param {Editor} editor
+     * @param {HTMLElement} el
+     */
+    static async getInlinedEditorContent(cssRulesByElement, editor, el) {
```

Comment:
in the commit message, the module should be "mail"

Reply:
Ok :+1:  Done